app/src/main/java/net/aiscope/gdd_app/ui/capture/CaptureImageActivity.kt

```diff
@@ -54,6 +55,23 @@ class CaptureImageActivity : AppCompatActivity(), CaptureImageView, CaptureFlow
                 cameraErrorCallback = { presenter.onCaptureError(it) }
             )
             zoomController = ZoomController(fotoapparat, cameraZoomLevel, cameraView)
+            toast  = Toast.makeText(
```

Comment:
This `toast` is going to hold a reference to this Activity, so it means that it needs to be unreferenced in `onDestroy()` to avoid memory leaks.

Reply:
I'm not sure how to do this properly, but take a look at my attempt